tests/ast/test_pre_parser.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from vyper.ast.pre_parser import validate_version_pragma
+from vyper.exceptions import VersionException
+
+
+def test_validate_version_pragma_valid(monkeypatch):
+    _mock_version(monkeypatch)
+    validate_version_pragma(' @version 0.1.0b17', (1, 0))
+    validate_version_pragma(' @version 0.1.3', (1, 0))
```

Comment:
Turn this into a parameterized test case:
```suggestion
@pytest.mark.parametrize('version', [
    vyper.__version__,
    '0.1.3',  # Maybe find some cleverer way of setting these,
    '1.0.0',  # so it's more scalable as we push new versions
])
def test_valid_version_pragma(monkeypatch, version):  # Shorter name too
    _mock_version(monkeypatch)
    validate_version_pragma(f' @version {version}', (1, 0))
```

Reply:
Also consider making `(1, 0)` into a readable constant like `SRC_LINE`

Reply:
Aye, Sir!